app/api/call_for_papers.py

```diff
@@ -56,17 +56,6 @@ def before_create_object(self, data, view_kwargs):
             event = self.session.query(Event).filter_by(id=view_kwargs['event_id']).one()
             data['event_id'] = event.id
 
-    def after_get(self, result):
```

Comment:
According to the discussion yesterday on gitter, as planned, we would need this function, right? To show the timezone naive dates by making it timezone aware?